core/metacling/src/TCling.cxx

```diff
@@ -1783,6 +1783,14 @@ void TCling::RegisterModule(const char* modulename,
       code += payloadCode;
 
    const char* dyLibName = nullptr;
+   // const char* dyLibName = FindLibraryName(triggerFunc);
+
+   // The triggerFunc may not be in a shared object but in an executable.
+   // In that case, we are done.
+   // bool isSharedLib = dyLibName && cling::DynamicLibraryManager::isSharedLibrary(dyLibName);
```

Comment:
What does that commented code line mean, is this part of the doc or a left-over?

Reply:
This is a WIP, trial and error development. This makes sure I remember what was done before...

Reply:
OK please label this as "do not review yet" or change the title to include [WIP] to let us know about the stage...